Initial check-in (#1)
im-open/create-github-deployment: README.md

# create-github-deployment

> This action creates GitHub Deployment and Deployment Status records.  It can be used in conjunction with tooling that reads GitHub deployments and deployment statuses to see the disposition of deployments.  This action defaults the `deployment.task` to `workflowdeploy` and is designed to work with an internal dashboard that utilizes custom payload data.

Reviewer: ```suggestion
This action creates GitHub Deployments and Deployment Statuses through [GitHub's API](https://docs.github.com/en/rest/deployments). This action defaults the `deployment.task` to `workflowdeploy` and is designed to work with an internal dashboard that utilizes custom payload data.
```
